charts/vela-core/values.yaml

```diff
@@ -81,4 +81,7 @@ admissionWebhooks:
 
 systemDefinitionNamespace: vela-system
 
+
+e2eTestEnable: false
```

Comment:
It's not good to add test related thing in user fields.

Reply:
right, I'll try to turn the e2e start option into a environment variable.